master/getting-started/kubernetes/installation/manifests/app-layer-policy/etcd/calico-networking/calico-node.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 layout: null
 ---
-{% helm %}
+{% helm calico-node.yaml %}
```

Comment:
FYI, this also outputs the service account, which is probably fine.

Reply:
Yes, but I believe that matches the original behavior before helm was introduced and mistakenly changed it.